# This patch fixes the issue mentioned here 
#https://www.drupal.org/project/drupal/issues/2951279
# It uses platform neutral UTF-8 encoding and \n newlines.
--- a/<html>Vid.php (<b>20 JAN 2021, 02:11:00</b>)</html>
+++ b/<html><b>Current File</b></html>
@@ -68,17 +68,13 @@
   public function titleQuery() {
     $titles = [];
 
-    $results = $this->database->query('SELECT nr.vid, nr.nid, npr.title FROM {node_revision} nr WHERE nr.vid IN ( :vids[] )', [':vids[]' => $this->value])->fetchAllAssoc('vid', PDO::FETCH_ASSOC);
-    $nids = [];
-    foreach ($results as $result) {
-      $nids[] = $result['nid'];
-    }
+    $results = $this->nodeStorage->getAggregateQuery()
+      ->allRevisions()
+      ->groupBy('title')
+      ->execute();
 
-    $nodes = $this->nodeStorage->loadMultiple(array_unique($nids));
-
     foreach ($results as $result) {
-      $nodes[$result['nid']]->set('title', $result['title']);
-      $titles[] = $nodes[$result['nid']]->label();
+      $titles[] = $result['title'];
     }
 
     return $titles;